api-docs/get-interactions-hours.md

```diff
@@ -0,0 +1,60 @@
+# Number of Interactions per Hour Range.
+
+Returns the total number of clients who have had at least that many client interaction hours in the given time period, along with the list of clients that match the specified criteria.
+
+## Request
+
+_The request should accept query parameters indicating a time range, number of client interaction hours, and current page._
+
+```http
+GET /api/interactions?start=2019-05-06T0&end=2019-05-08T0&=21600&hours=21600&page=1
+```
+
+Notes:
```

Comment:
Let's document each of the query params here - `start`, `end`, `minInteractionHours`, and `page`. You already have page documented, but let's document the rest

Reply:
The documentation should explain what the query param means, and also indicate what the default value is and whether it's a required param or not. Generally speaking, query params are optional - so it might be good to make all of the params have defaults.

If we add defaults to all the params, it also might be good to add to the response object some properties that explain what the params defaulted to:

```js
{
  "reportParameters": {
    "start": "2019-11-14" // some default
    ...
  }
}
```